### DISCUSSION TITLE
How to change the highlight within magit-diff-added-highlight

### DISCUSSION THREAD

**Discussion by jozuas** [magit/magit#4546] [Support] [answered]
![Screenshot 2021-11-26 at 11 50 34](https://user-images.githubusercontent.com/13085275/143561707-bad8420f-4cb8-4ee7-a73e-34a407fe92c1.png)

How can I change highlight colour within `magit-diff-added-highlight`? - I'd like to tone down the super bright green.

Thanks!

**Comment by tarsius** [comment, marked as answer]
Get started with `M-x customize-group RET magit-faces RET` and also see https://magit.vc/manual/magit/Theming-Faces.html.

**Reply by jozuas** [reply]
Sadly, this was not helpful. I cannot not find an option that allows me to customise the bright green within the added lines and the bright red within the removed lines.

To be extra specific, I'm referring to this:
![image](https://user-images.githubusercontent.com/13085275/143767151-b1833c8d-f61f-4bfe-9b24-da1bfd9bb3cc.png)

I had a look at `customize-group` and there was not a single option with bright green background and black font or a bright red background and grey font.

I have the following face config:

```elisp
   ;; Magit
   (magit-diff-added :background "#05180f" :foreground fg)
   (magit-diff-added-highlight :background "#052d18" :foreground fg)
   (magit-diff-removed :background "#330b04" :foreground fg)
   (magit-diff-removed-highlight :background "#780c00" :foreground fg)
   (magit-diff-context :background bg-alt :foreground fg)
   (magit-diff-context-highlight :background bg-alt :foreground fg)
   (magit-diff-hunk-heading :background bg-alt :foreground base3)
   (magit-diff-hunk-heading-highlight :background cyan :foreground bg :weight 'bold)
   (magit-diff-file-heading :foreground yellow :weight 'light)
   (magit-diff-file-heading-highlight :foreground yellow :weight 'bold)
   (magit-diff-file-heading-selection :foreground magenta :weight 'bold)
```

With or without this config, the bright green / bright red highlights remain.

**Reply by tarsius** [reply]
If you do `M-x customize-face`, then the face at point is offered as the default choice. Here it is `diff-refine-added`. I'll add those faces to the `magit-faces` group.

**Reply by jozuas** [reply]
Thank you so much!

I also want to thank you for magit because it's stunning.
